### DISCUSSION TITLE
Combine issue boards into one for phishing + phishing.database

### DISCUSSION THREAD

**Discussion by spirillen** [Phishing-Database/dev-center#9] [General]
In the past couple of days (2), and in relation to my attempt to generate a few issue templates, a thought have been flying around in my mind...

Is there at this point any good arguments for keeping to open and active issue boards for the same project? 

## Rationale

Phishing.Database is a automatically build and maintained list, build on scrapers, honeypots and it likes
Phishing is the manually applied datasets + the actual whitelist processing is going on

But most FP reports are posted in Phishing.Database and in race-condition to `https://github.com/Phishing-Database/phishing/blob/master/falsepositive*.list` using a subsidiary list `https://github.com/Phishing-Database/Phishing.Database/tree/master/whitelist.*`

This is programmable foolish to have 8 files to be imported, and you only need to import 4 from one repo, not 2. The files are build equally, with only slight diffs to the naming.

It is the same resources (Humans) that are commenting, approving/Rejecting, asking for additional information's if the issue are unclear, or something seems of.

Taking https://github.com/Phishing-Database/Phishing.Database/discussions/919 into account and consideration, it seems like the right thing to separate the whitelists from Phishing.Database to Phishing or maybe a third repo like Phishing.Exclusions, yet still only on board.

IF you/we/us are switching into one issue board, you disable PR for the public in Phishing.Database and Phishing, to avoid the race condition to reappear in another disguise

--------------

All these letters and thought, then I'm sure I've forgotten to take a thing or two into consideration, so please do just share your first thought, they are usual the bests.

Don't be scared or silent, any words counts towards the best solutions for all

**Comment by spirillen** [comment]
live example https://github.com/spirillen/Phishing-Database.WhiteLists

**Comment by funilrys** [comment]
done.
